e2e/playwright/test-utils.ts

```diff
@@ -961,7 +961,7 @@ export async function setup(
 
   await context.addCookies([
     {
-      name: COOKIE_NAME,
+      name: COOKIE_NAME_PREFIX + 'dev.zoo.dev',
```

Comment:
Is `dev.zoo.dev` meaningful here could it simply be some placeholder like `test`?

Reply:
I think we agreed in some other pr to use the base url domain.